src/main/resources/application.yml

```diff
@@ -44,6 +44,8 @@ nhais:
     mailboxPassword: ${NHAIS_MESH_MAILBOX_PASSWORD:password}
     sharedKey: ${NHAIS_MESH_SHARED_KEY:SharedKey}
     host: ${NHAIS_MESH_HOST:https://localhost:8829/messageexchange/} #local fake-mesh default endpoint
+    certValidation: ${NHAIS_MESH_CERT_VALIDATION:false}
```

Comment:
when certValidation is set to false in application.yml, then no need to add following variables to nhais-env-example.yaml:
NHAIS_MESH_CERT_VALIDATION
NHAIS_MESH_PUBLIC_SUFFIX
NHAIS_MESH_SUB_CA